.github/workflows/build-reuse-win.yml

```diff
@@ -85,15 +85,15 @@ jobs:
       uses: ilammy/setup-nasm@321e6ed62a1fc77024a3bd853deb33645e8b22c4
     - name: Prepare Machine
       shell: pwsh
-      run: scripts/prepare-machine.ps1 -ForBuild -Tls ${{ inputs.tls }}
+      run: scripts/prepare-machine.ps1 -ForBuild -Tls ${{ inputs.tls }} ${{ inputs.xdp }}
```

Comment:
We shouldn't need any build-specific stuff for XDP now. No special jobs. No special flags. Windows user just always builds in XDP.